internal/parse/parse.go

```diff
@@ -10,6 +10,26 @@ import (
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
+// Consumes mount flag in format of `--mount=type=bind,src=/path,from=image` and
+// return values of `src` and `from`, values are empty if keys are not present in the option.
+func GetFromAndSourceKeysFromMountFlag(mount string) (string, string) {
```

Comment:
This can be a helper function in the package that calls it until we find that we actually need to call it from somewhere else as well.